internal/pkg/log/log.go

```diff
@@ -33,26 +33,50 @@ type Logger interface {
 	//  ).Info("this is an error message")
 	With(args ...interface{}) Logger
 
-	// Log a message at level Debug, annotated with fields provided through With().
+	// Debug logs a message at level Debug, annotated with fields provided through With().
 	Debug(ctx context.Context, msg ...interface{})
 
-	// Log a message at level Info, annotated with fields provided through With().
+	// Info logs a message at level Info, annotated with fields provided through With().
 	Info(ctx context.Context, msg ...interface{})
 
-	// Log a message at level Warn, annotated with fields provided through With().
+	// Warn logs a message at level Warn, annotated with fields provided through With().
 	Warn(ctx context.Context, msg ...interface{})
```

Comment:
When do we want logging that is not formatted? 🤔 Are these methods still necessary?

Reply:
That is a good call out. Since the args are optional, `Warnf` would work where we're using the non-formatted methods currently. I'm going to consolidate these and keep the name of the current functions, `Warn`, `Error` etc. rather than their `<>f` equivalents.